packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-modal/add-patient-to-queue-entry.modal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, useState } from 'react';
+import { Button, Form, ModalBody, ModalFooter, ModalHeader, Stack, InlineNotification } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { usePatient, useVisit, type Visit } from '@openmrs/esm-framework';
+import { useMutateQueueEntries } from '../../hooks/useQueueEntries';
+import { type QueueEntry } from '../../types';
+import QueueFields from '../../create-queue-entry/queue-fields/queue-fields.component';
+import styles from './add-patient-to-queue-entry.scss';
+import capitalize from 'lodash/capitalize';
```

Comment:
Nit: please re-order the imports to follow - https://o3-docs.openmrs.org/docs/coding-conventions/code-organization:

> React and framework imports (e.g., React, useState, useEffect)
> External modules (e.g., lodash, dayjs, react-i8next)
> Carbon component imports (e.g., Button, InlineLoading)
> OpenMRS imports (e.g., @openmrs/esm-framework, @openmrs/esm-patient-common-lib)
> Local imports (components, hooks, utilities, etc.)
> Asset imports (e.g import styles from './user.scss')